impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -164,8 +164,8 @@ static constexpr vk::ImageUsageFlags ToVKImageUsageFlags(
   }
 
   if (mode != StorageMode::kDeviceTransient) {
-    // TODO (https://github.com/flutter/flutter/issues/121634):
-    // Add transfer usage flags to support blit passes
+    // Add transfer usage flags to support blit passes only if image isn't
```

Comment:
I found this outdated comment as I was looking for image usage flags. These flags are for allocated texture sources. The issue was with images that were wrapped swapchain images.